### DISCUSSION TITLE
gnet1.6.x linux-mips build失败(go version go1.18.3 darwin/arm64)

### DISCUSSION THREAD

**Discussion by lwwgo** [panjf2000/gnet#389] [Q&A] [answered]
命令：GOOS=linux GOARCH=mips go build  main.go。
在linux 、mac 上都试了下，均失败，报错如下：
# github.com/panjf2000/gnet/internal/netpoll
../../../go/pkg/mod/github.com/panjf2000/gnet@v1.6.5/internal/netpoll/defs_linux_mipsx.go:11:6: epollevent redeclared in this block
	../../../go/pkg/mod/github.com/panjf2000/gnet@v1.6.5/internal/netpoll/defs_default_linux.go:8:6: other declaration of epollevent

用-tags="linux,mips"也试过，报错一样。
辛苦看下是我编译命令有问题，还是bug，多谢！

**Comment by panjf2000** [comment, marked as answer]
拉一下 v2 的最新 commit: f2e2fa31，再试试。
